borsh/tests/schema/test_ip_addr.rs

```diff
@@ -0,0 +1,35 @@
+use crate::common_macro::schema_imports::*;
+
+#[cfg(feature = "std")]
+use std::net::IpAddr;
+
+#[test]
+fn ip_addr_schema() {
+    let actual_name = IpAddr::declaration();
+    let mut actual_defs = schema_map!();
+    IpAddr::add_definitions_recursively(&mut actual_defs);
+
+    assert_eq!("IpAddr", actual_name);
+    assert_eq!(
```

Comment:
this is a suggestion of how implementation for BorshSchema  may be added https://github.com/dj8yfo/borsh-rs/commit/7ee0dc90c7ad8ca477b63c57d8054b374db25408